content/src/main/content/jcr_root/apps/acs-commons/components/content/base/ajax/ajax.jsp

```diff
@@ -49,7 +49,7 @@
     if(StringUtils.isBlank(ajaxSelectors)) { ajaxSelectors = DEFAULT_SELECTOR; }
     if(StringUtils.isBlank(ajaxExtension)) { ajaxExtension = DEFAULT_EXTENSION; }
 
-    final String url = resourceResolver.map(resource.getPath()) + "." + ajaxSelectors + "." + ajaxExtension;
+    final String url = slingRequest.getContextPath() + resourceResolver.map(resource.getPath()) + "." + ajaxSelectors + "." + ajaxExtension;
```

Comment:
I think (but forget offhand) that this same thing can be done by calling resourceResolver(request, resource.getPath()). I think that's slightly cleaner. Can you try this out?